base/libgit2/utils.jl

```diff
@@ -43,3 +43,15 @@ function features()
     end
     return res
 end
+
+"""
+    LibGit2.posixpath(path)
+
+Standardise the path string `path` to use POSIX separators.
+"""
+function posixpath end
+if is_windows()
+    posixpath(path) = replace(path,'\\','/')
```

Comment:
for a unc network path this might be too simplistic, but I don't know whether libgit2's handling of this case is relevant here